### DISCUSSION TITLE
Specify python for fortls in vscode

### DISCUSSION THREAD

**Discussion by guziy** [fortran-lang/fortls#161] [Q&A] [answered]
Is it possible to configure python interpreter to be used by fortls?

I have system python 3.6 which is not compatible and I have fortls installed in a conda environment, I tried the following but it does not seem to work.

```json
{
    "fortran.fortls.path": "/path/to/fortls",
    "python.defaultInterpreterPath": "/path/to/python",
    
}
```

I am trying to use this in vscode-server through the remote-ssh.

thanks

**Comment by guziy** [comment, marked as answer]
I think I found a way to do it. 

Activate the conda environment then:

```bash
pip install --user fortls # wich will install fortls in ${HOME}/.local/bin with the python absolute path set in the sheebang
```

And specify the path to the pip version of `fortls` in remote settings of vscode, relaunch vscode and enjoy.

**Reply by gnikit** [reply]
It looks like you figured it out. What I was about to say is that if you have another version of Python just install `fortls` somewhere that will be reachable by your PATH and you should be good.
